tractor/_portal.py

```diff
@@ -102,14 +105,25 @@ async def aclose(self):
         """Cancel associated remote actor task and local memory channel
         on close.
         """
-        if self._rx_chan._closed:
+        rx_chan = self._rx_chan
+        stats = rx_chan.statistics()
+
+        if rx_chan._closed:
             log.warning(f"{self} is already closed")
             return
 
+        if stats.open_receive_channels > 1:
+            # if we've been cloned don't kill the stream
```

Comment:
I think you still need to call `rx_chan.close()` here?

Reply:
@richardsheridan nice catch. 

Yeah i feel like you're right.
I wonder why I didn't notice this being a problem (yet).